__tests__/developers.test.js

```diff
@@ -110,5 +97,18 @@ describe('Developers', () => {
       const res = await getWord(limitWordId, { apiLimit: 2 }, { apiKey: developerRes.body.apiKey });
       expect(res.status).toEqual(403);
     });
+
+    it('should return developer document with correct credentials', async () => {
+      const developerRes = await createDeveloper(developerData);
+      const developerDetails = await getDeveloper({ apiKey: developerRes.body.apiKey });
+      expect(developerDetails.status).toEqual(200);
+      expect(developerDetails.body).not.toEqual(undefined);
```

Comment:
@davydocsurg instead of checking to see if the body is undefined, can you check to see if the body includes the correct data?

Reply:
Alright